WordPress/src/main/res/layout/my_site_blogging_prompt_card_avatar_item.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="wrap_content"
+    android:layout_height="match_parent">
+
+    <ImageView
+        android:backgroundTint="?attr/colorSurface"
+        android:id="@+id/avatar"
+        style="@style/MySiteCardBloggingPromptAvatar"
```

Comment:
⚠️  This makes the avatar small, but it seems it's being used in the generic "base" `AvatarViewHolder` and `TrainOfAvatarsItem` which are also used in the Reader post details. See other comments.

I think it's good to have created this new layout file with the specific dashboard needs, but I feel like we also need to duplicate the `AvatarViewHolder` and add a way for setting which ViewHolder the `TrainOfAvatarsAdapter` will use depending on the screen/component using it (Reader vs Dashboard Card).